railties/lib/rails/generators/erb/mailer/templates/layout.html.erb.tt

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
+    </style>
```

Comment:
The layout file of new rails app contains empty style tags. 
https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/app/views/layouts/mailer.html.erb.tt#L5..L7 

Should not make it the same?